.github/workflows/release.yaml

```diff
@@ -59,7 +59,7 @@ jobs:
     steps:
       - name: Set version
         id: version
-        run: echo ::set-output name=version::${GITHUB_REF#refs/*/}
+        run: echo version=${GITHUB_REF#refs/*/} >> $GITHUB_OUTPUT
```

Comment:
Should all of these be surrounded by quotes to avoid any space related issues?

Reply:
TBH I have no idea, I can add quotes it shouldn't hurt.

Reply:
Fixed.